File: app/src/main/java/com/idnoffice/app/FcmTokenHelper.java
package com.idnoffice.app;
import android.util.Log;
import com.google.firebase.messaging.FirebaseMessaging;
public class FcmTokenHelper {

    public static String latestToken = null;
    public static void getToken() {
        FirebaseMessaging.getInstance().getToken()
                .addOnSuccessListener(token -> {
                    latestToken = token;
                    Log.d("FCM_TOKEN", "Token ni bozz: " + token);
                })
                .addOnFailureListener(e -> {
                    Log.e("FCM_TOKEN", "Gagal ambil token", e);
                });
    }
}

File: app/src/main/java/com/videa/app/FcmTokenHelper.kt
package com.videa.app

import android.util.Log
import com.google.firebase.messaging.FirebaseMessaging

object FcmTokenHelper {
    @JvmStatic
    var latestToken: String? = null

    @JvmStatic
    fun getToken() {
        FirebaseMessaging.getInstance().token
            .addOnSuccessListener { token ->
                latestToken = token
                Log.d("FCM_TOKEN", "Token ni bozz: $token")
            }
            .addOnFailureListener { e ->
                Log.e("FCM_TOKEN", "Gagal ambil token", e)
            }
    }
}